templates/repo/header.tmpl

```diff
@@ -5,7 +5,8 @@
 			<div class="column"><!-- start column -->
 				<div class="ui header">
 					<div class="ui huge breadcrumb">
-						<i class="mega-octicon octicon-{{if .IsPrivate}}lock{{else if .IsMirror}}repo-clone{{else if .IsFork}}repo-forked{{else}}repo{{end}}"></i>
+						{{if .RelAvatarLink|len}}<img class="ui mini spaced image" src="{{.RelAvatarLink}}">{{else}}<i class="mega-octicon octicon-repo"></i>{{end}}
```

Comment:
No need to pipe `|len`, same for other places.

Reply:
Nope. Without `|len` it doesn't work.
Even empty string not counted as `false`.

Reply:
See https://golang.org/pkg/text/template/#hdr-Actions

Reply:
Huh. Now it's work as expected.
I'll remove `|len`.
Don't know why it "glitched".

Reply:
Done